General.md

```diff
@@ -0,0 +1,6 @@
+# General Standards at ASH
+
+## Commenting
+
+Comments should be placed on the line above the code that it describes.
+> Why? When tracking change history in a version control system (in our case, TFS) by changesets associated with line numbers, it is easier to see when changes were applied to a comment vs actual code.
```

Comment:
i would remove the TFS comment. It's unnecessary and teams are starting to talk about the possibility of git